recipes/simbac/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "simbac" %}
+{% set version = "0.1a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tbrown91/SimBac/archive/50158978df536e10e4fcb2f9cd3871deb6da99ca.zip
+  sha256: 82d42a4aef46c22c3d6472c1f3bf4915f6461bc7ec6eabca2bf1a3e9e3c2a914
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
```

Comment:
A cpp compile is enough isn't it?

Reply:
I tested it and it is. I have applied the modification.